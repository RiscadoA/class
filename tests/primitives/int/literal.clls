proc main() {
    println(0);
    println(1);
    println(2);
    println(9);
    println(10);
    println(11);
    println(10000);
    println(99999);
    println(-1);
    println(-50);
    println(-452942);
    println(32598279);
    ()
};;
