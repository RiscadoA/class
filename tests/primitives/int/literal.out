0
1
2
9
10
11
10000
99999
-1
-50
-452942
32598279
